modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -1,317 +1,460 @@
+--
+The percentage of the total memory allocated to the CloudHub workers that run the application, including the memory consumed by the JVM itself, reaches a capacity threshold for a specified period.
 
+image::alerts-conditions-apps-memory.png[Memory usage - Cloudhub]
 
-* *Secure data gateway disconnected*
-* *Secure data gateway connected*
-* *Worker not responding*
-* *Deployment success*
-* *Deployment failure*
+To enable this condition, specify the capacity threshold and duration that triggers the alert.
+--
 
-=== Conditions on Locally Deployed Applications
 
-On an application that's deployed to local servers, both via the xref:deployment-strategies.adoc#deployment-scenarios[cloud console or the Anypoint Platform Private Cloud Edition], you can select any of these alert conditions:
+The following alerts for applications deployed to CloudHub require no additional parameters:
 
-* *Number of errors*
+* Deployment failed
+* Deployment success
+* Secure data gateway connected
+* Secure data gateway disconnected
+* Worker not responding
 
-+
-image::alerts-on-runtime-manager-387e2.png[]
 
-+
-You must provide the number of errors that triggers the alert.
-The time unit is per minute. An alarm is triggered when the error count goes over the threshold number in the timeslot of one minute.
-+
-[NOTE]
-This alert is currently not available on the xref:deployment-strategies.adoc#deployment-scenarios[Anypoint Platform Private Cloud Edition].
+=== Conditions for Applications Deployed to Local Servers 
 
-* *Number of Mule messages*
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-+
-image::alerts-on-runtime-manager-51033.png[]
+For applications deployed to local servers, the following alert conditions apply:
 
+Number of errors::
 +
-You must provide the number of Mule messages, since the application was started, that trigger the alert.
+--
+The number of errors in one minute reaches a threshold.
 
-+
-[NOTE]
-This alert is currently not available on the xref:deployment-strategies.adoc#deployment-scenarios[Anypoint Platform Private Cloud Edition].
+image::alerts-conditions-number-errors.png[Number of errors]
 
-* *Response time*
-+
-image::alerts-on-runtime-manager-45785.png[]
+To enable this condition, specify the number of errors per minute that triggers the alert.
 
+--
+
+Number of Mule messages::
 +
-You must provide the maximum accepted response time--anything over this value triggers the alert.
+--
+The number of Mule messages, since the application started, reaches the specified limit.
```

Comment:
```suggestion
The number of Mule messages since the application started reaches the specified limit.
```
The parenthetical is actually not; it is a required part of the subject.